im-sdk-core/src/main/java/com/easemob/im/server/api/block/room/join/BlockUserJoinRoom.java

```diff
@@ -0,0 +1,50 @@
+package com.easemob.im.server.api.block.room.join;
+
+import com.easemob.im.server.api.Context;
+import com.easemob.im.server.exception.EMUnknownException;
+import com.easemob.im.server.model.EMBlock;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+public class BlockUserJoinRoom {
+
+    public static Flux<EMBlock> getBlockedUsers(Context context, String roomId) {
```

Comment:
Please change it to non-static method, and move the Context into instance member.